src/components/WalletNavigator.js

```diff
@@ -110,7 +110,7 @@ const WalletTabNavigator = injectIntl(
       isReadOnly: any,
     }) => (
       <Tab.Navigator
-        initialRouteName={WALLET_ROUTES.TX_HISTORY}
+        initialRouteName={WALLET_ROUTES.DASHBOARD}
```

Comment:
What happens if it's a Byron wallet (for which the `DASHBOARD` tab won't be rendered)?

Reply:
Tab.Navigator will default to the next route, but I think is better to make it explicit, so I'll be fixing it.